headers/types/dungeon_mode/dungeon_mode.h

```diff
@@ -101,7 +101,11 @@ struct statuses {
     uint8_t burn_turns;  // 0x17: Turns left for the status in statuses::burn
     // 0x18: Turns left until residual damage for the status in statuses::burn, if applicable
     uint8_t burn_damage_countdown;
-    undefined field_0x19;
+    // 0x19: The number of times the statuses::burn_damage_countdown has reached 0. Only used
+    // when badly poisoned. Determines how much damage the badly poisoned status condition
+    // will deal. There is no noticable difference because the table this value is looked up
+    // on is filled with 0x6
+    uint8_t burn_damaged_count;
```

Comment:
If it's used for badly poisoned pokémon, then why is it called `burn_damaged_count`?